docs/user-guide/archived-items.md

```diff
@@ -22,19 +22,37 @@ Because <span class="status-warning">:bootstrap-x-octagon-fill: Canceled</span>
 
 ## Archived Item Details
 
-The archived item details page is composed of five sections, though the Crawl Settings tab is only available for crawls and not uploads.
+The archived item details page is composed of six sections, though some tabs are only available for crawls and not uploads.
 
 ### Overview
 
 The Overview tab displays the item's metadata and statistics associated with its creation process.
 
 Metadata can be edited by pressing the pencil icon at the top right of the metadata section to edit the item's description, tags, and collections it is associated with.
 
+### Quality Assurance
+
+The Quality Assurance tab displays a list of crawled pages, overall stats collected through crawl analysis, and a list of all analysis runs. This tab is not available for uploaded archived items.
```

Comment:
This could lead more with how "list of crawled pages" and "overall stats" is relevant to a QA process. Maybe just by bringing up some of the keywords in the second paragraph earlier? Something like:

```diff
+ The Quality Assurance tab displays quality assurance data that is collected from manual reviews and analysis runs. This is where you can start new analysis runs, view quality metrics from older runs, and delete runs.

The pages list provides a record of all pages within the archived item, as well as any ratings or notes given to the page during review. If analysis has been run, clicking on a page in the pages list will go to that page in the review interface.
```